Source/Core/VideoCommon/RenderState.h

```diff
@@ -22,11 +22,18 @@ union RasterizationState
 {
   void Generate(const BPMemory& bp, PrimitiveType primitive_type);
 
-  RasterizationState& operator=(const RasterizationState& rhs);
+  RasterizationState() = default;
```

Comment:
Not sure if we should be declaring the move semantics as well so that we can be explicit with expectations?

Reply:
(same with `FramebufferState` )

Reply:
I've added them, but they feel like a lot of boilerplate for something that doesn't actually behave differently when moved.